rs/tests/nns/BUILD.bazel

```diff
@@ -160,7 +160,9 @@ system_test(
 
 system_test(
     name = "bitcoin_set_config_proposal_test",
-    env = NNS_CANISTER_ENV,
+    env = dict(NNS_CANISTER_ENV.items() + {
+        "BTC_WASM_PATH": "$(rootpath @btc_canister//file)",
+    }.items()),
```

Comment:
Em, why not just add your new entry to NNS_CANISTER_ENV?

For better or worse, it must be that your WASM is already available to the test somehow (I guess system_test adds a data dep). If so, doing as I suggested ☝️ seems like it would not bloat the dependency graph. IIUC, it would just give other tests an environment variable that they don't use.

Anyway, I do not have a strong opinion about this -> approve.